src/controllers/expense.controller.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const expenseService = require('./../services/expense.service');
+
+const get = async(req, res) => {
+  res.send(expenseService.get());
+};
+
+const getOne = async(req, res) => {
+  const { id } = req.params;
+  const expense = expenseService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = async(req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expense = expenseService.create(req.body);
+
+  res.status(201);
+  res.send(expense);
+};
+
+const update = async(req, res) => {
+  const { id } = req.params;
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  if (!expenseService.getById(id)) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  const expense = expenseService.update(id, req.body);
+
+  res.send(expense);
+};
+
+const remove = async(req, res) => {
+  const { id } = req.params;
+
+  if (!expenseService.getById(id)) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  const newExpense = expenseService.remove(id);
+
+  res.sendStatus(204);
+  res.send(newExpense);
```

Comment:
the `remove` method returns undefined. so what did you trying to send to the client?